app/customer.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+
+with open("app/config.json", "r") as j_file:
+    customer = json.load(j_file)["customers"]

```

Comment:
you should load all data before using it and avoid overusing the context manage in another places